README.md

```diff
@@ -41,32 +40,28 @@ Below is the list of variables you can redefine in your playbook to customize st
 | `mistral_db_password` | `StackStorm`  | PostgreSQL DB password for Mistral.
```

Comment:
Looks like big room for improvement for number of variables/settings.
For now OK as something minimum viable.